### DISCUSSION TITLE
Put badges in one single line

### DISCUSSION THREAD

**Discussion by prishs** [adityatelange/hugo-PaperMod#1275] [Q&A]
Is there a way we can put code status badges side by side in hugo papermod.
Is it possible? every badge is rendering on different line

**Comment by RoneoOrg** [comment]
Which badges are you talking about?
Could you please add a screenshot or an URL for the demo?

**Reply by prishs** [reply]
using badges from shields.. 
[![GitHub Repo stars](https://img.shields.io/github/stars/mkdocs/mkdocs)]()
[![GitHub contributors](https://img.shields.io/github/contributors/mkdocs/mkdocs)]()
[![GitHub commit activity (branch)](https://img.shields.io/github/commit-activity/m/mkdocs/mkdocs)]()

these are rendering in separate lines in hugo website..
![image](https://github.com/adityatelange/hugo-PaperMod/assets/31505174/a3e4753c-5f98-441d-a70e-cbb00b415958)

Expecting something like this
![image](https://github.com/adityatelange/hugo-PaperMod/assets/31505174/548c51b0-542d-4c8c-8bba-e631d0fef400)

**Reply by prishs** [reply]
was able to get it work using html inline styles.. thanks..

**Reply by RoneoOrg** [reply]
Glad to hear that @prish-collab ! Remember to close this issue to keep the tracker clean
